### ISSUE TITLE
why the training loss decreased very slowly?

### ISSUE THREAD

**Issue by xuxiang0408** [xiaodiu2010/TextBoxes-TensorFlow#12]
When I trained on SynthText data, the loss decreased very slowly and vibrated near 6.0. I tried to decrease the lr, but no help. Any suggestion?

**Comment by xiaodiu2010** [comment]
Can you list details in your experiment, batch size, lr , etc...? thanks.

**Comment by hellotf** [comment]
batch size = 8, lr = 0.0001 and use_batch = True

**Comment by xiaodiu2010** [comment]
batch_size it too small. you may try bs = 32 or higher. If you have limited power to do this, you may low your lr to 0.0001/4 or smaller.

**Comment by hellotf** [comment]
OK, I'll try the smaller lr. Thx:)

**Comment by hellotf** [comment]
my machine power is too weak....Is it possible that you give a link for your pretrained model on synthtext data?
